### DISCUSSION TITLE
Notifications question

### DISCUSSION THREAD

**Discussion by LilTrublMakr** [netbox-community/netbox#9650] [Q&A]
Is there a way to create a popup/toast notification in Netbox using a Python script? I have done a little Googling and have only found native support for email notifications. If not, not a big deal since I can use Apprise.

These are the ones I am talking about:
![Screenshot 2022-07-01 233601](https://user-images.githubusercontent.com/54646041/176985180-5b3346f9-ccbe-4ba3-83c7-4daa8bc29ac9.png)

**Comment by jeremystretch** [comment]
These are how NetBox currently displays any messages sent via [Django's messages framework](https://docs.djangoproject.com/en/4.0/ref/contrib/messages/). So calling `messages.info()` for example will display the provided message as a notification.

**Reply by LilTrublMakr** [reply]
I will look in to this during the weekend. Does this have to be within Netbox/plugins or can this be used in an outside python script and sent to Netbox? I am trying to do the latter.

**Reply by qwertycandy** [reply]
@LilTrublMakr Hello - sorry to bother you, but did you get those messages to work? If so, would you mind sharing an example of your code, please? I'm dealing with the same problem and I can't figure it out. Thank you :)

**Comment by quique** [comment]
Besides Django's messages framework, `project-static/src/bs.ts` allows us to use Bootstrap's Toast (see commit 0ea9c650 ).

So you could do something like

```js
const toast = new Toast(toastElement, { delay: 3000 });
toast.show();
```
